### PR TITLE
Adds Search to App

### PR DESCRIPTION
**Pull request by redreceipt**
## DESCRIPTION

Screen Shot 2019-10-15 at 2 53 25 PM

### What does this PR do, or why is it needed?

Adds search bar to the discover feed.

### How do I test this PR?

Go to the discover feed with and without `EXPERIMENTAL=true`

## TODO

- [x] I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))
- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android if applicable
- [x] Set a relevant reviewer

## REVIEW

- [ ] Review updates to test coverage and snapshots
- [ ] Review code through the lens of being concise, simple, and well-documented

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by richarddubay** [comment]
@redreceipt If you want to hop in here and resolve these conflicts I can start reviewing this one.

**Comment by redreceipt** [comment]
resolved

File: packages/newspringchurchapp/src/tabs/connect/Connect.js
Comment (richarddubay) [inline packages/newspringchurchapp/src/tabs/connect/Connect.js]: Not sure why this change is in here, but I'm pretty sure we can get rid of it now since we're showing the `ActionBar` again.

**Review by richarddubay — CHANGES_REQUESTED** [review]

**Comment by richarddubay** [comment]
@redreceipt I need to test it, but from a code perspective this looks pretty good. Do you see value in getting the latest from Core for these files (some have been updated since you pulled them) before we merge? Or is just waiting for the next Core update good enough?

File: packages/newspringchurchapp/src/tabs/connect/Connect.js
Comment (redreceipt) [reply packages/newspringchurchapp/src/tabs/connect/Connect.js]: yep you're right

Comment (redreceipt) [inline packages/newspringchurchapp/src/tabs/connect/Connect.js]:
```suggestion
            <ActionBar />
```

**Comment by redreceipt** [comment]
> @redreceipt I need to test it, but from a code perspective this looks pretty good. Do you see value in getting the latest from Core for these files (some have been updated since you pulled them) before we merge? Or is just waiting for the next Core update good enough?

I'd say go with what we have an we'll pick up the updates later

**Comment by richarddubay** [comment]
I can't prove it works, but it certainly looks like it should. Time to pony up some 💰for Algolia it looks like.

**Comment by richarddubay** [comment]
Marked this as blocked waiting on an Algolia decision.

**Comment by redreceipt** [comment]
This should be good to go now right?

—
Michael

**Comment by richarddubay** [comment]
Yep! I actually started looking at it again. We'll need to clean up the merge conflicts once we get the new core update merged in.

Also, I discovered this while testing: https://github.com/ApollosProject/apollos-prototype/issues/1197

**Comment by richarddubay** [comment]
@redreceipt When you get some time, can you take a look at this branch again? The app doesn't even want to load for me any more. It just loads the splash screen and then stays there.
Screenshot 2019-11-05 15 34 38

**Comment by redreceipt** [comment]
i reverted this back to a working version and it's showing conflicts again so i guess something weird did happen when i merged it earlier

**Comment by redreceipt** [comment]
i think i fixed it

**Comment by redreceipt** [comment]
not sure why Travis is failing here, tests are passing locally for me

**Review by richarddubay — APPROVED** [review]

**Pull request merged**
